src/service/ParticipantService.ts

```diff
@@ -121,7 +121,7 @@ ParticipantService.Router.delete("/participant/:participant_id", async (req: Req
 })
 ParticipantService.Router.get("/participant/:participant_id", async (req: Request, res: Response) => {
   try {
-    let output = { data: ParticipantService.get(req.get("Authorization"), req.params.participant_id) }
+    let output = { data:await ParticipantService.get(req.get("Authorization"), req.params.participant_id) }
```

Comment:
There should be a space after the colon for consistent formatting.

Reply:
Ok. We used to format it @avaidyam . This time, I have quickly done without any formatting. We will take care next time

Reply:
No problem for now. Perhaps we should integrate eslint and prettier into the build/run phase like was done for the frontend code. (Linting issues will cause warnings/errors and fail to build the image.)

Reply:
Ok. We can do it